projects/igniteui-angular/src/lib/directives/mask/mask.directive.ts

```diff
@@ -57,24 +49,22 @@ export class IgxMaskDirective implements OnInit, ControlValueAccessor {
      * ```html
      * <input placeholder = "enter text...">
      * ```
-     * @memberof IgxMaskDirective
      */
+    @DeprecateProperty('"placeholder" is deprecated, use native placeholder instead.')
```

Comment:
Yeah this will likely trigger deprecation warnings along with normal attribute assignment due to it being an Input.
Might need to find a better way to handle this.